elixir/apps/domain/priv/repo/migrations/20230926211345_update_clients_upstream_dns_column_type.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Domain.Repo.Migrations.UpdateClientsUpstreamDnsColumnType do
+  use Ecto.Migration
+
+  # def change do
+  #  alter table("configurations") do
+  #    modify(:clients_upstream_dns, {:array, :map},
+  #      from: {:array, :string},
+  #      default: [],
+  #      null: false
+  #    )
+  #  end
+  # end
+
+  @drop_column "ALTER TABLE configurations DROP COLUMN clients_upstream_dns;"
```

Comment:
No need to move it to the function attribute, just inline the SQL into `execute/1`.

Reply:
Yeah, I had originally tried something a little different and meant to discuss this with you synchronously.  I'll get this moved in to the `execute/1` function if you're ok with handling the migration this way.